packages/kit/src/core/adapt/builder.js

```diff
@@ -181,7 +181,10 @@ export function create_builder({
 		},
 
 		writeClient(dest) {
-			return copy(`${config.kit.outDir}/output/client`, dest);
+			const files = copy(`${config.kit.outDir}/output/client`, dest);
+			// avoid making vite files public
+			rimraf(`${dest}/.vite`);
+			return filter_vite_files(files);
```

Comment:
I think rather than deleting and filtering it out I would just avoid copying it in the first place. Perhaps `copy` could take an additional arg that's a filter?

Reply:
Good idea. I missed that there already was one. Might need to tweak it a bit to get it to work here.